test/framework/src/main/java/org/elasticsearch/AnalysisFactoryTestCase.java

```diff
@@ -178,7 +175,7 @@ private static String toCamelCase(String s) {
         .put("apostrophe",                ApostropheFilterFactory.class)
         .put("arabicnormalization",       ArabicNormalizationFilterFactory.class)
         .put("arabicstem",                ArabicStemTokenFilterFactory.class)
-        .put("asciifolding",              ASCIIFoldingTokenFilterFactory.class)
+        .put("asciifolding",              Void.class)  // TODO remove this when core no longer depends on analysis-common
```

Comment:
I'll push something that makes this a bit more clear/fancy. It'll be temporary while we do the migration, but it'll help.